scripts/linters/pylint_extensions_test.py

```diff
@@ -2234,3 +2234,253 @@ def test_file_overview_at_end_of_file(self):
+                    class ClassName(dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+
+
+                        a = 1 + 2
+                """)
+        node_excessive_newline_below_class_docstring.file = filename
+        node_excessive_newline_below_class_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_excessive_newline_below_class_docstring)
+
+        message = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            line=6)
+
+        with self.checker_test_object.assertAddsMessages(message):
+            temp_file.close()
+
+    def test_class_with_no_docstring(self):
+        node_class_with_no_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        def func():
+                            \"\"\"This is a docstring.\"\"\"
+                            a = b + c
+                """)
+        node_class_with_no_docstring.file = filename
+        node_class_with_no_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_class_with_no_docstring)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
+
+    def test_class_with_no_body(self):
+        node_class_with_no_body = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+
+                    def func():
+                        a = 1 + 2
+                """)
+        node_class_with_no_body.file = filename
+        node_class_with_no_body.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_class_with_no_body)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
+
+    def test_decorated_function_after_class_docstring(self):
+        node_decorated_function_after_class_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(dummy_class):
+                        \"\"\"This is a docstring.\"\"\"
+                        @decorator()
```

Comment:
Use real situation make it @staticmethod

Reply:
Done.